Sources/Framework/Extensions/UIView/UIView+Rounded.swift

```diff
@@ -0,0 +1,6 @@
+extension UIView {
+    func rounded() {
```

Comment:
If you make it a verb it will be easier to understand that it mutates the state. For example makeRoundedCorner/applyRoundedCorners.